ports/stm32/spi.c

```diff
@@ -47,22 +47,22 @@
 // SPI6_RX: DMA2_Stream6.CHANNEL_1
 
 #if defined(MICROPY_HW_SPI1_SCK)
-SPI_HandleTypeDef SPIHandle1 = {.Instance = NULL};
+static SPI_HandleTypeDef SPIHandle1 = {.Instance = NULL};
```

Comment:
should we use `STATIC` instead of `static` to be consistent with the rest of the file?

Reply:
yes, please use `STATIC` for consistency

Reply:
Oops, fixed. Thanks!